ping.go

```diff
@@ -0,0 +1,107 @@
+// Package ping provides a facility for pinging hosts that uses the installed ping command
+// rather than Go's ICMP support, enabling basic pinging without needing administrator/root
+// privileges.
+package ping
+
+import (
+	"bufio"
+	"bytes"
+	"io"
+	"os/exec"
+	"strconv"
+
+	"github.com/getlantern/errors"
+)
+
+const (
+	// DefaultCount is 1
+	DefaultCount = 1
+
+	// DefaultPayloadSize is 1472 bytes
+	DefaultPayloadSize = 1472
```

Comment:
Why make this the default size?  It's pretty non-standard from what I can tell.

Reply:
It results in a packet that's the same size as the typical Ethernet MTU of 1500.

Reply:
Right.  Is that something typically desired in sending pings?

Reply:
Yeah, not really. I'll switch to the default Unix value of 56.